onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/IgnoredAlert.java

```diff
@@ -1,3 +1,18 @@
+/**
+ * Copyright (C) 2023 MBTA <lboyarsky@mbta.com>
```

Comment:
Haha, thank you 😆

Reply:
lol, one of the build validations makes sure that every file has this header and fails if it's missing.